locale/en/docs/es6.md

```diff
@@ -55,12 +55,26 @@ All ES6 features are split into three groups for **shipping**, **staged**, and *
 
 * [Arrow Functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
 
+* [new.target](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new.target)
+
+* [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe)
```

Comment:
Agree @mzgol 

@evanlucas this should go to ES2016 (Is there a list of ES2015/ES2016 that v8 supports? This could be in sync with that list)